_layouts/download.html

```diff
@@ -65,7 +65,7 @@ <h3>CircuitPython {{ version.version }}</h3>
             </label>
             <div class="download-buttons">
             {% for file in version.files['en_US'] %}
-              <a class="download-button-{{stable}}" href="{{ file }}">DOWNLOAD {{ file | slice: -4, 4 | upcase }} NOW <i class="fas fa-download"></i></a>
+              <a class="download-button {% if version.stable %}stable{% else %}unstable{% endif %} {{ file | slice: -3, 3 }}" href="{{ file }}">DOWNLOAD {{ file | slice: -4, 4 | upcase }} NOW <i class="fas fa-download"></i></a>
```

Comment:
Does this need a `-` after `download-button` to match the old version?

Reply:
Is it used anywhere? I didn't find a reference to it anywhere.

Reply:
Not that I know of. Just thought it was since it was in there.